Source/WebCore/style/values/StyleValueTypes.h

```diff
@@ -511,6 +607,16 @@ template<typename StyleType, size_t inlineCapacity> struct Blending<SpaceSeparat
         }
         return true;
     }
+    auto canBlend(const SpaceSeparatedVector<StyleType, inlineCapacity>& a, const SpaceSeparatedVector<StyleType, inlineCapacity>& b, const RenderStyle& aStyle, const RenderStyle& bStyle) -> bool
+    {
+        if (a.size() != b.size())
+            return false;
+        for (size_t i = 0; i < a.size(); ++i) {
```

Comment:
Zipped range?

Reply:
Do we have a usable zippered range? I think the c++ ranges one is unavailable to us for now due to some macOS and iOS builds still using a relatively old libc++ (which didn't implement all the ranges features until later).

Reply:
There's a file `ZippedRange.h`. The word _zippered_ lost some letters, it seems.